apps/condo/migrations/20221013103705-0177_organizationemployeerole_cancreateemployees_and_more.js

```diff
@@ -0,0 +1,85 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Add field canCreateEmployees to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canCreateEmployees" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canCreateEmployees" DROP DEFAULT;
+--
+-- Add field canDeleteEmployees to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canDeleteEmployees" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canDeleteEmployees" DROP DEFAULT;
+--
+-- Add field canUpdateEmployees to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canUpdateEmployees" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canUpdateEmployees" DROP DEFAULT;
+--
+-- Add field canCreateEmployees to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canCreateEmployees" boolean NULL;
+--
+-- Add field canDeleteEmployees to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canDeleteEmployees" boolean NULL;
+--
+-- Add field canUpdateEmployees to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canUpdateEmployees" boolean NULL;
+
+-- set canCreateEmployees to true and canManageEmployees to false for Foreman employee role
```

Comment:
1. no need for 3 updates, 1 should be enough
2. use CAPS for SQL commands like SELECT, SET, WHERE, ...
3. no DOWN migration updates reverting these changes
4. looks like this updates should be moved to scripts.